scripts/lc_classifiers_letters_only.json

```diff
@@ -0,0 +1,232 @@
+  "JC": "Political theory",
+  "JF": "Political institutions and public administration",
+  "JJ": "Political institutions and public administration (North America)",
+  "JK": "Political institutions and public administration (United States)",
+  "JL": "Political institutions and public administration (Canada, Latin America, etc.)",
+  "JN": "Political institutions and public administration (Europe)",
+  "JQ": "Political institutions and public administration (Asia, Africa, Australia, Pacific Area, etc.)",
+  "JS": "Local government. Municipal government",
+  "JV": "Colonies and colonization. Emigration and immigration. International migration",
+  "JX": "International law, see JZ and KZ (obsolete)",
+  "JZ": "International relations",
+  "K": "Law",
+  "KB": "Religious law in general. Comparative religious law. Jurisprudence",
+  "KBM": "Jewish law",
+  "KBP": "Islamic law",
+  "KBR": "History of canon law",
+  "KBS": "Canon law of Eastern churches",
+  "KBT": "Canon law of Eastern Rite Churches in Communion with the Holy See of Rome",
+  "KBU": "Law of the Roman Catholic Church. The Holy See",
+  "KD/KDK": "United Kingdom and Ireland",
+  "KDZ": "America. North America",
+  "KE": "Canada",
+  "KF": "United States",
+  "KG": "West Indies. Caribbean area",
+  "KH": "South America",
+  "KJ-KKZ": "Europe",
+  "KL-KWX": "Asia and Eurasia, Africa, Pacific Area, and Antarctica",
+  "KU/KUQ": "Law of Australia and New Zealand",
+  "KZ": "Law of nations",
+  "L": "Education",
+  "LA": "History of education",
+  "LB": "Theory and practice of education",
+  "LC": "Special aspects of education",
+  "LD": "United States",
+  "LE": "America (except United States)",
+  "LF": "Europe",
+  "LG": "Asia, Africa, Indian Ocean islands, Australia, New Zealand, Pacific islands",
+  "LH": "College and school magazines and papers",
+  "LJ": "Student fraternities and societies, United States",
+  "LT": "Textbooks",
+  "M": "Music",
+  "ML": "Literature on music",
+  "MT": "Instruction and study",
+  "N": "Fine Arts",
+  "NA": "Architecture",
+  "NB": "Sculpture",
+  "NC": "Drawing. Design. Illustration",
+  "ND": "Painting",
+  "NE": "Print media",
+  "NK": "Decorative arts",
+  "NX": "Arts in general",
+  "P": "Language and Literature",
+  "PA": "Greek language and literature. Latin language and literature",
+  "PB": "Modern languages. Celtic languages and literature",
+  "PC": "Romanic languages",
+  "PD": "Germanic languages. Scandinavian languages",
+  "PE": "English language",
+  "PF": "West Germanic languages",
+  "PG": "Slavic languages and literatures. Baltic languages. Albanian language",
+  "PH": "Uralic languages. Basque language",
+  "PJ": "Oriental languages and literatures",
+  "PK": "Indo-Iranian languages and literatures",
+  "PL": "Languages and literatures of Eastern Asia, Africa, Oceania",
+  "PM": "Hyperborean, Native American, and artificial languages",
+  "PN": "Literature (General)",
+  "PQ": "Portuguese literature",
```

Comment:
According to LOC, this should be French, Italian, Spanish, and Portuguese literature. https://www.loc.gov/aba/cataloging/classification/lcco/lcco_p.pdf